Attachement reminder

lu-kaf opened the pull request:
It would be nice if we could have a message saying 'you forgot to attach your attachment' if there are some specific keywords in the message. 

Some webmails such as roundcube already have plugins doing that, but there is not that possibility on RainLoop / snappymail.

the-djmaze commented:
Can you close this?
It is spamming my notifications feed

the-djmaze commented:
@lu-kaf or @RainLoop  please kill this issue. It is flooding my notifications log.

Commit:
allow multi-account in nc with oauth login

akhil1508 committed:
Fix capabilities when THREAD is disabled

Signed-off-by: Akhil <akhil@e.email>

akhil1508 committed:
Merge branch 'master' of https://github.com/Murena-SAS/snappymail into dev/nc-oauth-multiaccount

akhil1508 committed:
Also check if the account has been defined in setting

akhil1508 committed:
Also perform logout if explicit creds change

Commit:
Make #1689 optional through application.ini

the-djmaze committed:
Merge pull request #1698 from Murena-SAS/dev/fix-set-capabilities

Fix capabilities when THREAD is disabled

the-djmaze committed:
Merge pull request #1699 from Murena-SAS/dev/nc-oauth-multiaccount

allow multi-account in nc with oauth login

Commit:
Drop `md5(file_get_contents(DATA.php))` because i think nobody used this undocumented feature

Commit:
Don't use focusedMessage for active menu items, it could confuse

Commit:
FileStorage strict properties

Commit:
Improved handling of RFC 2822 dates #1694

Commit:
Use PHP_SAPI

Commit:
Bugfix: Attempt to read property "smimeSigned" on null

Commit:
v2.37.0

Commit:
Bugfix: Gulp v5 broke copying fonts.
https://github.com/gulpjs/gulp/issues/2797

Commit:
v2.37.1

Commit:
Validate Fetch JSON response for #1703

Commit:
Prevent logout loop on error

Commit:
Show content-type error on login

Commit:
Resolve #1703

Commit:
v2.37.2 for Nextcloud

Commit:
Resolve #1645

Commit:
v2.37.2

Artoria2e5 committed:
Update zh, zh-TW translations

This commit updates zh and zh-TW translations. I try to match the language used in the file, but I still hope a native speaker from the other side of the strait can check my work.

Commit:
Resolve #1710

Commit:
Change handling of (token) errors due to #1706

Commit:
Resolve #1709

the-djmaze committed:
Merge pull request #1712 from Artoria2e5/zh-update

Update zh, zh-TW translations

Commit:
Added editing of ACL rules #157

Commit:
Small UI fix for #157

Commit:
Small cleanup of #157

Commit:
Resolve #1574

Commit:
Resolve #1678

Commit:
Improved domain autoconfig interaction

Commit:
Added turndown.js for #1604

Commit:
Simplify sieve scripts list #1675

Commit:
Parse error #1712

Commit:
Localization strings for ACL #157

Commit:
Fix CRLF #1712

Commit:
Cleanup Cacher handling and added some documentation

Commit:
Added markedjs for #1604

hguilbert committed:
Update user.json

the-djmaze committed:
Merge pull request #1717 from hguilbert/patch-28

Update user.json

Commit:
Resolve #1706

Commit:
Mark images with width=1 as tracking pixel

Commit:
Resolve #1718

Commit:
Make sure only scalar values are allowed in $_ENV #1719

Commit:
Show GnuPG version for #1560

Commit:
MS autodiscover priorities DNS over subdomain

Commit:
Show warning in Admin -> About when PHP runs in 32bit

Commit:
Sabre/Xml v4.0.5

Commit:
Force Nextcloud personal language by default #1428

Commit:
v2.37.3

ner00 committed:
Update Portuguese

ner00 committed:
SETTINGS_GENERAL/MAILTO

Commit:
Bugfix: Call to a member function Email() on null #1706

the-djmaze committed:
Merge pull request #1722 from ner00/master

Update Portuguese

Commit:
Resolve #1724

Commit:
Resolve #1725

Commit:
Bugfix: application/pgp-encrypted attachments not listed

Commit:
Improved clear cookie handling

Commit:
Bugfix: `use_local_proxy_for_external_images` is part of `labs`, not `security`
@SonarSource

Commit:
Debug Turndown ES2020 #1604

Commit:
Added option to convert HTML to Markdown instead of plain, using Turndown #1604

hguilbert committed:
Update user.json

the-djmaze committed:
Merge pull request #1749 from hguilbert/patch-29

Update user.json

Commit:
RTL language improvement from #1744 by @rezaei92

Commit:
Use Squire 2.3.2 from https://github.com/the-djmaze/Squire/commits/snappymail/

Commit:
Squire bugfixes

Commit:
Improved SquireUI handling of pathChange and bugfix `bold` match

Commit:
Resolve #1715

Commit:
Update Squire with SnappyMail changes

Commit:
Implemen #1737 including an i18n fix

Commit:
Resolve #1746

Commit:
Bump version number

Commit:
Improved attempt for #1746

Commit:
Resolve #1733

Commit:
Also prevent error for #1733

Commit:
Updated custom Squire 2.3.2

Commit:
cleanHtml use allowedTags instead of disallowedTags and improved CSS handling

the-djmaze committed:
Update SECURITY.md

Commit:
v2.38.0

Commit:
Resolve #1754

Commit:
Use the custom Squire as submodule vendors/squire2

Commit:
PHP 8.4: Implicitly nullable parameter declarations deprecated

PhieF committed:
new plugin for ics view, based on ical lib

the-djmaze committed:
Merge pull request #1745 from Murena-SAS/dev/new-view-ics-plugin

Display ICS preview using an ICS library

Commit:
Bugfix: event.organizer could be undefined #1745

Commit:
Replaced ical.es5.min.cjs with ical.js v2.1.0 for #1745

Commit:
Bugfix: Admin - Config `search` should be ko.observable() not ko.observableArray()

Commit:
Added Admin - Extensions search filter

Commit:
Improved ical parsing for "view-ics" extension

bilogic committed:
fix en

bilogic committed:
fix lv

the-djmaze committed:
Merge pull request #1759 from bilogic/case-correction

Case correction

Commit:
Change for #1706 to debug more

Commit:
Bugfix: ClientException #1706

Commit:
Added mapping for sSmtpUser

Commit:
Fix extensions with `login.credentials` hook parameters

Commit:
Resolve #1716

Commit:
Changes for #1706

ner00 committed:
Update Portuguese

the-djmaze committed:
Merge pull request #1768 from ner00/master

Update Portuguese

Commit:
Resolve #1761

Commit:
Resolve #1726

hguilbert committed:
Update user.json

Commit:
Some small idea changes

Commit:
Bugfix for #1709

Commit:
Resolve #1770

Commit:
fix spacing

tinola committed:
Update user.json

the-djmaze committed:
Merge pull request #1771 from tinola/master

Update user.json

the-djmaze committed:
Merge pull request #1769 from hguilbert/patch-30

Update user.json

Commit:
Improve sieve parser

Commit:
Improved Sieve parser validation

Commit:
Fix Sieve QuotedString handling

Commit:
Some Sieve parser improvements

Commit:
Improved Knockout property binding error

Commit:
Fix Sieve math-type and comparator handling

Commit:
KnockoutJS don't throw error when property is not observable

Commit:
Keep 1 space between HTML elements

Commit:
KnockoutJS remove `with($context)` scope for faster access

Commit:
Resolve #1746

Commit:
Drop support for noembed and noframes
Also remove everything after `<` inside `<style>`

Commit:
Added extension info to Admin -> Extensions -> Popup

Commit:
O365 option to use Azure or Personal accounts #1645

Commit:
Resolve #1780

Commit:
Bugfix: composer dialog scroll got broken in v2.28

Commit:
Bugfix: Composer dialog "from" triangle wrong position due to font changes

Commit:
Squire: prevent collapse of empty elements so that the caret can be placed in there.
This eliminates the use of hacky <br> inside.

Commit:
Squire sanitizeToDOMFragment use cleanHTML

Commit:
Fixed the translation of the fields MIDDLE_NAME, NAME_PREFIX, NAME_SUFFIX to pt-BR

Commit:
Squire: improved handling of BR elements

Commit:
Bugfix the new msgId handling due to Squire sanitizeToDOMFragment using cleanHTML

the-djmaze committed:
Merge pull request #1785 from mstolf/bugfix_translate_ptBr

Fixed the translation of the fields MIDDLE_NAME, NAME_PREFIX, NAME_SU…

Commit:
Resolve #1787

Commit:
Merge branch 'master' of github.com:the-djmaze/snappymail

tinola committed:
Update user.json

update Polish translation

tinola committed:
Update admin.json

update Polish translation (admin page)

Commit:
In Order to https://github.com/nextcloud/server/pull/45819 this change will adjust boarders to the new smaller boarder radius.

the-djmaze committed:
Merge pull request #1788 from tinola/master

update Polish translations

Commit:
Remove ""

Signed-off-by: Patrick Niebeling <patrick.niebeling@adacor.com>

the-djmaze committed:
Merge pull request #1790 from gnilebein/45819--Adjust-Boarder-Radius

Reduce border radius element to 8 px

Commit:
Squire paste image working again.
https://github.com/the-djmaze/snappymail/issues/1389#issuecomment-2389678680

Commit:
Merge branch 'master' of github.com:the-djmaze/snappymail

Commit:
Resolve #1765

Commit:
Resolve #1775

Commit:
Remove #1737

Commit:
Added the proper logging for #1706

SergeyMosin committed:
make sure that KO bindings are re-applied

Commit:
Prepare for #1793

SergeyMosin committed:
use 'Utils.cleanHtml' as pasteSanitizer + minor tweaks

SergeyMosin committed:
bump version to v1.0.5

Commit:
Resolve #1706

the-djmaze committed:
Merge pull request #1796 from SergeyMosin/compact-composer-plugin-v1.0.5

Compact composer plugin v1.0.5

Commit:
v2.38.1

Commit:
Resolve #1797

Commit:
Secure ko template bindings?

Commit:
v2.38.2

Commit:
Use CSS instead of $index #1797

Commit:
KnockoutJS secure bindings using Proxy

Commit:
Resolve #1798

Commit:
Added setting for default messages sort mode

hguilbert committed:
Update user.json

tinola committed:
Update user.json

Update Polish translation

the-djmaze committed:
Merge pull request #1800 from tinola/master

Update user.json

the-djmaze committed:
Merge pull request #1799 from hguilbert/patch-31

Update user.json

romawkafor committed:
Update user.json

I propose changes for the Ukrainian language, taking into account syntax and untranslated phrases.

In a short time, I will try to create and translate an admin page (admin.json) and 2-Step Verification (TOTP), which I will also offer

romawkafor committed:
Create uk.ini

Created a localization file for the Ukrainian language for the password change application.

romawkafor committed:
Create uk.ini

Created a localization file for the Ukrainian language for the two-factor authentication application

romawkafor committed:
Update cs.ini

Removed the typo in lines 15 and 16 (the \n symbol at the end of the tape)

romawkafor committed:
Update en.ini

Removed the typo in lines 15 and 16 (the \n symbol at the end of the tape)

romawkafor committed:
Update hu.ini

Removed the typo in lines 15 and 16 (the \n symbol at the end of the tape)

romawkafor committed:
Update it.ini

Removed the typo in lines 15 and 16 (the \n symbol at the end of the tape)

romawkafor committed:
Update zh.ini

Removed the typo in lines 15 and 16 (the \n symbol at the end of the tape)

romawkafor committed:
Create admin.json

I am adding the possibility to use the Ukrainian language for the admin panel

romawkafor committed:
Update user.json

I am adding some changes

the-djmaze committed:
Merge pull request #1810 from romawkafor/master

Corrections and addition of Ukrainian language localization files

romawkafor committed:
Update user.json

Minnor changes

Commit:
Resolve #1806

Commit:
Merge branch 'master' of github.com:the-djmaze/snappymail

Commit:
Bugfix: sortMode undefined at login

Commit:
Cleanup knockout a bit

Commit:
Proper fix for default sortMode

Commit:
Fix Knockout binding non-Observable notice

Commit:
Improved some templates

Commit:
IdentityModel extends EmailModel

Commit:
On reply, set `from` field to the received email `to` field #1805

the-djmaze committed:
Merge pull request #1816 from romawkafor/master

Please update user.json in Ukraine language.

ner00 committed:
Update Portuguese

the-djmaze committed:
Merge pull request #1822 from ner00/master

Update Portuguese

Commit:
Resolve #1815

Commit:
Resolve #1813

Commit:
Rename Identity.formattedName to Identity.toString

Commit:
Improve contacts layout

Commit:
Speedup handling of identical avatars

Commit:
Speedup identicons by caching them

Commit:
Set identicon in cache when loading remote avatar fails

Commit:
Preload identicon on delayed avatar lookup

Commit:
Added test contacts CardDAV settings

Commit:
Put knockout bind context Proxy outside bindingFunction

Commit:
ko.utils.domNodeDisposal.addDisposeCallback to ko.addDisposeCallback

Commit:
Cleanup KnockoutJS a bit

Commit:
Resolve #1828

Commit:
Resolve #1825

Commit:
Only allow Proxy for logged in users

edlerd committed:
Add dmarc and spf indicators in mail view #1820

Signed-off-by: David Edler <david.edler@canonical.com>

SergeyMosin committed:
update squire to v2.3.2 and fix willPaste listener/sanitizer

Commit:
Improvements for #1846

the-djmaze committed:
Merge pull request #1845 from SergeyMosin/compact-composer-plugin-v1.0.6

Update squire to v2.3.2 and fix willPaste listener/sanitizer

edlerd committed:
Show icon if no information is avilable on dkip, dmark or spf

Signed-off-by: David Edler <david.edler@canonical.com>

Commit:
Resolve #1850

Commit:
Resolve #1849

the-djmaze committed:
Merge pull request #1843 from edlerd/add-spf-dmarc

Add dmarc and spf indicators in mail view

lenny87 committed:
Finished czech translation of Snappymail

lenny87 committed:
Fixed typo in Spam folder name

hungyao committed:
Update Html.js to remove x-html tags before rendering

Fix for https://github.com/the-djmaze/snappymail/issues/1860 to remove <x-html></x-html> tags that might be emitted by certain email clients.

Tested by modifying locally using JS debugger and Snappymail JS debug mode.

the-djmaze committed:
Merge pull request #1861 from hungyao/patch-1

Update Html.js to remove x-html tags before rendering

the-djmaze committed:
Merge pull request #1857 from lenny87/cs-translation

Finished czech translation of Snappymail

Commit:
Also detect Identity using `from` address #1865

bapt committed:
fr: add an space before the colon

When composing a message in reply to an email, a colon is appended to
the message which is translated, in french colon should always be
prepended by an unbreakable space.

Add a space in the translation sentence to make sure the space appears
before the colon in the interface

tribut committed:
Fix incorrect German translation

HeySora committed:
Automatically add favicon host to CSP if present

simeongoranov committed:
Fix DateTimeHelper::ParseRFC2822DateString integer overflow on 32-bit PHP

TomsProject committed:
Changing hashing algorithm to sha256

RChutchev committed:
Update user.json

Commit:
Resolve #1930

Commit:
Resolve #1883

the-djmaze committed:
Merge pull request #1918 from TomsProject/master

Changing Gravatar hashing algorithm to sha256

Commit:
Bump version for #1918

the-djmaze committed:
Merge pull request #1891 from HeySora/master

Automatically add favicon host to CSP if present

the-djmaze committed:
Merge pull request #1880 from tribut/i18n-de-fixes

Fix incorrect German translation

RChutchev committed:
Merge branch 'the-djmaze:master' into fix-ru-lang

RChutchev committed:
Update user.json

bober757 committed:
incorrect using of the preposition

bober757 committed:
incorrect using of the preposition

Commit:
openssl_pkey_free is deprecated

makoehr committed:
More compact view of folder list

With many folders, the line-height is too large.
It is reduced to make the view more compact.
Also, the top margin for the number of unread-folder will
be adapted.

Psychi1 committed:
Fix Cp1252 encoding

Psychi1 committed:
fix regex match-group

the-djmaze committed:
Merge pull request #1961 from Psychi1/Cp1252

Fix the Cp1252 encoding problem

the-djmaze committed:
Merge pull request #1948 from makoehr/compact-folder-list-for-snappymail

More compact view of folder list

Commit:
Small change for #1948 to make it relative to font size

the-djmaze committed:
Merge pull request #1941 from bober757/master

incorrect using of the preposition

the-djmaze committed:
Merge pull request #1931 from TLTDevRu/fix-ru-lang

Fix ru lang

the-djmaze committed:
Merge pull request #1875 from bapt/fr

fr: add an space before the colon

the-djmaze committed:
Merge pull request #1914 from sgoranov/fix/1913-datetime-overflow

Fix DateTimeHelper::ParseRFC2822DateString integer overflow on 32-bit PHP

Commit:
Catch Throwable for #1914

Commit:
Ommit Nextcloud "max-version" as it is optional
